src/elm/Search.elm

```diff
@@ -0,0 +1,478 @@
+                        [ text <| t "menu.search.overviewFound"
+                        , text " "
+                        , strong []
+                            [ text (String.fromInt count)
+                            , text " "
+                            , text <|
+                                if count == 1 then
+                                    singular
+
+                                else
+                                    plural
+                            ]
+                        ]
+                    ]
+                , button
+                    (class "button w-auto button-sm px-6"
+                        :: (if count == 0 then
+                                [ class "button-disabled" ]
+
+                            else
+                                [ class "button-primary"
+                                , onClick showMsg
+                                ]
+                           )
+                    )
+                    [ text (t "menu.search.overviewShow") ]
+                ]
+    in
+    div []
+        [ strong [ class "block py-4" ]
+            [ text (t "menu.search.overviewHeader") ]
+        , ul []
+            [ viewItem Icons.shop
+                (List.length offers)
+                (t "menu.search.offer")
+                (t "menu.search.offers")
+                (TabActivated OffersTab)
+            , viewItem Icons.flag
+                (List.length actions)
+                (t "menu.search.action")
+                (t "menu.search.actions")
+                (TabActivated ActionsTab)
+            ]
+        ]
+
+
+viewOffers : Symbol -> List Offer -> Html Msg
+viewOffers symbol offers =
+    let
+        viewOffer : Offer -> Html Msg
+        viewOffer offer =
+            li
+                [ class "flex px-2 w-1/2 sm:w-1/3 md:w-1/4" ]
+                [ div
+                    [ class "rounded-md overflow-hidden bg-white flex-grow mb-4 pb-4 cursor-pointer hover:shadow"
+                    , onClick (FoundItemClicked (Route.ViewSale (String.fromInt offer.id)))
+                    ]
+                    [ case offer.image of
+                        Nothing ->
+                            text ""
+
+                        Just url ->
+                            img [ src url ] []
+                    , h3 [ class "p-3" ] [ text offer.title ]
+                    , p [ class "px-3 leading-none" ]
+                        [ span [ class "text-xl text-green font-medium" ] [ text <| String.fromFloat offer.price ]
+                        , br [] []
+                        , span [ class "text-gray-300 text-xs" ]
+                            [ text <| Eos.symbolToSymbolCodeString symbol
+                            ]
+                        ]
+                    ]
+                ]
+    in
+    ul [ class "offers-list flex flex-wrap mt-6 mb-8 mx-2 justify-left" ]
+        (List.map viewOffer offers)
+
+
+isActive : Model -> Bool
+isActive model =
+    case model.state of
```

Comment:
maybe this can be a boolean expression?

Reply:
> maybe this can be a boolean expression?

I don't think so... Here are some details.

The search panel could be in different states:

```elm
type State
    = Inactive
    | Loading
    | RecentSearchesShowed
    | ResultsShowed (Maybe SearchResult) ActiveTab
```

The `Inactive` variant represents the state when the search form is not used and the search results are invisible. This `isActive` function just checks if we have the `Inactive` state so we can check it and show the other page blocks instead.

I need to pass the `Search.Model` into `Search.isActive` because I want to use it from other modules.